engine/utils/generate_visitors/README.md

```diff
@@ -0,0 +1,14 @@
+# `generate_visitors`
+
+This binary reads the AST module of hax and creates **standalone**
```

Comment:
This certainly needs to go into the top level dev.md.

Reply:
I added some more high level documentation in DEV.md!